eng/scripts/Language-Settings.ps1

```diff
@@ -219,20 +219,21 @@ function Update-java-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$nu
 
 
 # function is used to filter packages to submit to API view tool
-function Find-java-Artifacts-For-Apireview($artifactDir, $pkgName = "")
+function Find-java-Artifacts-For-Apireview($artifactDir, $pkgName)
 {
-  Write-Host "Checking for source jar in artifact path $($artifactDir)"
+  $packageSearchPath = Join-Path $artifactDir "*" $pkgName
```

Comment:
Is this wildcard sufficient? For example, will this handle the scenario where I have two different sets of package files in the same directory, e.g.

```
my-package-1.0.0.jar
my-package-foo-1.0.0.jar
```

We do artifact segregation in our pipelines presently so it should be OK, but its worth just considering any corner cases here.

Reply:
It might be worth asserting exactly the file set we expect.